[2023-03-29 13:29:18,774] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T13:28:56.910609+00:00 [queued]>
[2023-03-29 13:29:18,814] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T13:28:56.910609+00:00 [queued]>
[2023-03-29 13:29:18,816] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:29:18,817] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 13:29:18,819] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:29:18,856] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): transform_data_task> on 2023-03-29 13:28:56.910609+00:00
[2023-03-29 13:29:18,866] {standard_task_runner.py:52} INFO - Started process 13742 to run task
[2023-03-29 13:29:18,874] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v2', 'transform_data_task', 'manual__2023-03-29T13:28:56.910609+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpu1q5ei1g', '--error-file', '/tmp/tmppwce7odc']
[2023-03-29 13:29:18,877] {standard_task_runner.py:77} INFO - Job 109: Subtask transform_data_task
[2023-03-29 13:29:19,002] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T13:28:56.910609+00:00 [running]> on host 5ebc180c7cc0
[2023-03-29 13:29:19,147] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v2
AIRFLOW_CTX_TASK_ID=transform_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T13:28:56.910609+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T13:28:56.910609+00:00
[2023-03-29 13:31:00,412] {stock_data_transform.py:132} ERROR - An error occurred during transformation: Recursive view `stock_data` detected (cycle: `stock_data` -> `stock_data`)
[2023-03-29 13:31:00,418] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 13:31:00,564] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v2, task_id=transform_data_task, execution_date=20230329T132856, start_date=20230329T132918, end_date=20230329T133100
[2023-03-29 13:31:00,748] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 13:31:00,971] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
